packages/core/utils/src/sourcemap.js

```diff
@@ -30,15 +30,18 @@ export async function loadSourceMapUrl(
   if (match) {
     let url = match[1].trim();
     let dataURLMatch = url.match(DATA_URL_RE);
-    filename = dataURLMatch ? filename : path.join(path.dirname(filename), url);
+    let mapFilePath = dataURLMatch ? filename : url.replace('file://', '');
```

Comment:
```suggestion
    let mapFilePath = dataURLMatch ? filename : url.replace(/^file:///, '');
```

or something along those lines?

Reply:
Sure, dont expect file:// in the middle of the filepath but better to be safe I guess